navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -138,6 +144,24 @@ private void initialize() {
     }
   }
 
+  private void initializeTelemetry() {
+    validateAccessToken(accessToken);
+    String sdkIdentifier = MAPBOX_NAVIGATION_SDK_IDENTIFIER;
+    if (isFromNavigationUi) {
+      sdkIdentifier = MAPBOX_NAVIGATION_UI_SDK_IDENTIFIER;
+    }
+    String userAgent = String.format("%s/%s", sdkIdentifier, BuildConfig.MAPBOX_NAVIGATION_VERSION_NAME);
+    MapboxTelemetry.getInstance().initialize(context, accessToken, userAgent, sdkIdentifier,
+      BuildConfig.MAPBOX_NAVIGATION_VERSION_NAME);
+    MapboxTelemetry.getInstance().newUserAgent(userAgent);
```

Comment:
Should we take care of setting the user agent for the UI here as well (now that we know if we are initializing from the UI or not)?  Rather than when I initialize in `NavigationView` https://github.com/mapbox/mapbox-navigation-android/blob/master/navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java#L428

Reply:
@danesfeder we're already doing it 👀
https://github.com/mapbox/mapbox-navigation-android/blob/1867c75d693f9f47a0fa4abae8a369edeb089391/navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L148